### ISSUE TITLE
833 & Exception Attribute New Installation Errors Solved

### ISSUE THREAD

**Issue by awolfa** [inkstitch/inkstitch#824]
I am not sure the technical terms of my problem.  I just loaded Ink Stitch, to the folder required.  I am attaching pictures to show the problem instead of trying to sadly explain 
![MainScreen](https://user-images.githubusercontent.com/33202696/93886328-45e1e880-fcb3-11ea-9cab-159d06fb92b7.PNG)
![Second Screen](https://user-images.githubusercontent.com/33202696/93886351-4b3f3300-fcb3-11ea-836d-0873522bca61.PNG)

**Comment by awolfa** [comment]
This issue was solved.  Here is trail, just for continuity if someone else runs into the problem:


Window Version BTW--

1.  I followed the Video Instructions.

 On the last part of the video it has you test with a t with a shape :  Object to Path - Ext -Inkst - Params

1.  I installed as the video indicated. -- User Extensions.  (photo's were included) - Issue of it not working. -- GRR

2. I went with the 2nd option and copied the files direct to the InkStitch Extensions.  *Note I said copied  -- Got the error message 833. 

3.  Deleted the files out of User Extension.  Got Exception Attribute Error..

4. Remembered I had downloaded the file on my desktop and opened the file before watching the video.  Deleted all the files.

PRESTO - It works.
